src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.ImportBefore.targets

```diff
@@ -31,36 +29,18 @@
   <Import Condition=" $(SonarQubeTargetsImported) != 'true' AND $(BuildingInsideVisualStudio) != 'true' AND Exists('$(SonarQubeTargetFilePath)') " Project="$(SonarQubeTargetFilePath)" />
 
   <!-- **************************************************************************** -->
-  <!-- Diagnostic/error-checking target
-
-        Causes the build to fail if the analysis properties are not correctly configured
-        i.e. if the user has requested a SonarQube analysis run, but the analysis targets
-        cannot be found.
-
-        Writes out diagnostic information to help with troubleshooting.  -->
+  <!-- Write out diagnostic information to help with troubleshooting.               -->
   <!-- **************************************************************************** -->
   <Target Name="SonarQubeImportBeforeInfo" Condition=" $(SonarQubeTempPath) != '' AND $(BuildingInsideVisualStudio) != 'true' " BeforeTargets="CoreCompile">
 
     <PropertyGroup>
       <AnalysisTargetsFileFound Condition=" Exists('$(SonarQubeTargetFilePath)') " >true</AnalysisTargetsFileFound>
-      <ReportAnalysisTargetsError Condition=" $(AnalysisTargetsFileFound) != 'true'">true</ReportAnalysisTargetsError>
     </PropertyGroup>
 
     <!-- Diagnostic messages for troubleshooting -->
     <Message Importance="normal" Text="Sonar: ($(MSBuildProjectName)) SonarQube.Integration.ImportBefore.targets was loaded" />
-    <Message Importance="low" Text="Sonar: ($(MSBuildProjectFile)) SonarQube analysis targets file found: $(AnalysisTargetsFileFound)" />
```

Comment:
Why remove this debug log?

Reply:
It's redundant information.
When found, we won't be investigating here and we'll see many other messages in the log indicating the presence of the imported target.
When not found - well have the `Importance="high"` message below.